RHEL/6/input/xccdf/system/software/integrity.xml

```diff
@@ -266,4 +267,77 @@ computer viruses, as well as to limit their spread to other systems.
 
 </Group>
 
+<Group id="fips">
+<title>Federal Information Processing Standard (FIPS)</title>
+<description>
+The Federal Information Processing Standard (FIPS) is a computer security standard which
+is developed by the U.S. Government and industry working groups to validate the quality
+of cryptographic modules. The FIPS standard provides four security levels to ensure
+adequate coverage of different industries, implementation of cryptographic modules, and
+organizational sizes and requirements.
+<br /><br />
+FIPS 140-2 is the current standard for validating that mechanisms used to access cryptographic modules
+utilize authentication that meets industry and government requirements. For government systems, this allows
+Security Levels 1, 2, 3, or 4 for use on Red Hat Enterprise Linux.
+<br /><br />
+See <b>http://csrc.nist.gov/publications/PubsFIPS.html</b> for more information.
+</description>
+
+<Rule id="package_dracut-fips_installed" severity="medium">
+<title>Install the dracut-fips Package</title>
+<description>
+To enable FIPS, the system requires that the <tt>dracut-fips</tt>
+package be installed. To install the <tt>dracut-fips</tt> package:
+<pre>$ sudo yum install dracut-fips</pre>
+</description>
+<ocil clause="the package is not installed">
+<package-check-macro package="dracut-fips" />
+</ocil>
+<rationale>
+Use of weak or untested encryption algorithms undermines the purposes of utilizing encryption to
+protect data. The operating system must implement cryptographic modules adhering to the higher
+standards approved by the federal government since this provides assurance they have been tested
+and validated.
+</rationale>
+<ident cce="RHEL6-CCE-TBD" />
+<oval id="package_dracut-fips_installed" />
+<ref nist="AC-17(2)" disa="68,2450" />
+</Rule>
+
+<Rule id="grub_enable_fips_mode" severity="medium">
+<title>Enable FIPS Mode in GRUB</title>
+<description>
+To ensure all FIPS mode is enabled, rebuild <tt>initramfs</tt> by run the following command:
+<pre>dracut -f</pre>
+After the <tt>dracut</tt> command has been run, add the argument <tt>fips=1</tt> to the default
+GRUB kernel line for the Linux operating system in
+<tt>/etc/grub.conf</tt>, in the manner below:
+<pre>kernel /vmlinuz-version ro vga=ext root=/dev/VolGroup00/LogVol00 rhgb quiet fips=1</pre>
+</description>
+<ocil clause="FIPS is not configured or enabled in grub">
+To verify that FIPS is enabled properly in grub, run the following command:
+<pre>$ grep fips /etc/grub.conf</pre>
+The output should contain fips=1
+</ocil>
+<rationale>
+Use of weak or untested encryption algorithms undermines the purposes of utilizing encryption to
+protect data. The operating system must implement cryptographic modules adhering to the higher
+standards approved by the federal government since this provides assurance they have been tested
+and validated.
+</rationale>
+<warning category="general">Running <pre>dracut -f</pre> will overwrite the existing initramfs file.</warning>
```

Comment:
Here again would suggest to add another warning that the system needs to be rebooted for the changes to take effect.